src/components/HeroSection/index.js

```diff
@@ -23,7 +23,14 @@ const HeroSection = () => {
   return (
     <HeroContainer>
       <HeroBg>
-        <VideoBg autoPlay loop muted src={Video} type="video/mp4"></VideoBg>
+        <VideoBg
+          autoPlay
+          playsInline
+          loop
+          muted
+          src={Video}
+          type="video/mp4"
+        ></VideoBg>
```

Comment:
If a react component doesn't have anything between the tags you can just close the first tag. Ex. <VideoBg autoPlay playsInLine ... />